websites/D/Discord SFX/presence.ts

```diff
@@ -70,6 +70,19 @@ presence.on("UpdateData", async () => {
 		];
 	}
 
+	if (pathname.startsWith("/users/")) {
+		presenceData.details = `Viewing ${
+			document.title.split(" Profile")[0].split(" | ")[1]
+		} profile`;
+
+		const { searchParams } = new URL(
+			document.querySelector<HTMLImageElement>('[data-premid-id="avatar"]')
+				?.src ||
+				"https://cdn.rcd.gg/PreMiD/websites/D/Discord%20SFX/assets/logo.png"
+		);
+		presenceData.largeImageKey = searchParams.get("url");
```

Comment:
This still has the same issue. I was wrong in my initial request. If the image element is not found, it would've errored out, rather than resulting in an empty largeImageKey. Either add some null checks, or use a `try/catch` block

Reply:
I understand what you're saying. I'll make a few changes. Also keep in mind there won't be a situation where the image element can't be found. There will always be an image element on the page

Reply:
I see. I was assuming that there could be a chance that there is no image, since you were adding a null check on the `.src` property. If the image is always there, then
```suggestion
		const { searchParams } = new URL(
			document.querySelector<HTMLImageElement>('[data-premid-id="avatar"]')
				.src
		);
		presenceData.largeImageKey = searchParams.get("url");
```
might be more clear.

Reply:
I did the null check because prettier told me to do it